ddclient.in

```diff
@@ -7162,6 +7175,65 @@ sub nic_keysystems_update {
   }
 }
 
+######################################################################
+## nic_regfishdev4_examples
+######################################################################
+sub nic_regfishdev4_examples {
+    return <<EoEXAMPLE;
+o 'regfishde'
+The 'regfishde' protocol is used by the non-free dynamic DNS service offered by www.regfish.de.
+Check https://www.regfish.de for configuration variables applicable to the 'regfishde' protocol:
+protocol=regfishde
+server=dyndns.regfish.de
+password=service-password          ## password (token) registered with the service
+myhost.mydomain.com                ## the host registered with the service.
+Example ${program}.conf file entries:
+## single host update
+protocol=regfishde, \
+password=service-password \
+myhost.mydomain.com
+EoEXAMPLE
+}
+
+######################################################################
+## nic_regfishdev4_update
+## response contains "success" on succesfull completion
+######################################################################
+sub nic_regfishdev4_update {
+ debug("\nnic_regfishdev4_update -------------------");
+
+ ## update each configured host
+ ## should improve to update in one pass
+ for my $h (@_) {
+   my $ip = delete $config{$h}{'wantip'};
+   info("regfish.de setting IP address to %s for %s", $ip, $h);
+
+   my $url = "https://$config{$h}{'server'}/?fqdn=$h&ipv4=$ip&forcehost=1&token=$config{$h}{'password'}";
```

Comment:
What is with ipv6? Should we set forcehost all the time?

Reply:
IPv6 is on Beta. For the moment it does not work as simple as IPv4.
It is no problem to use forcehost all the time but it would also be got to have the choice if use it or not but i don't know how so i inserted it as static enabled.

Reply:
> IPv6 is on Beta. For the moment it does not work as simple as IPv4 but i will stay tuned.

Sure, what is complicated about it?



> Forcehost means that if the DNS-entry does not exist to autocreate it, if forcehost is disabled it have to be created manually.

That is the behavior in most other providers, too and prevents typos.

Reply:
> Sure, what is complicated about it?
You're right, someone wrote that it's complicated but opposite the manual the url remeains the same, changes only &ipv4= to &ipv6= so the url would be:
`$y $url = "https://$config{$h}{'server'}/?fqdn=$h&ipv4=$ip&forcehost=$config{$h}{'forcehost'}&token=$config{$h}{'password'}";`
But how should this best be implemented? With a own protocol or with a if ... then ... else?

> That is the behavior in most other providers, too and prevents typos.
So would my suggestion be right? If yes i will integrate the code.
By the way thanks for support

Reply:
Please merge.